.github/workflows/build-and-store-wasm.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
 
       # Upload the WASM bundle as an artifact
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
           name: wasm-bundle
           path: src/wasm-lib/pkg
```

Comment:
Syntax should remain, see others already migrated like https://github.com/KittyCAD/modeling-app/blob/83f6c4767a70bcf768e72d66faff93277737b243/.github/workflows/build-apps.yml#L187